jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/common/BreadcrumbTests.java

```diff
@@ -0,0 +1,139 @@
+package io.github.epam.bootstrap.tests.common;
+
+import com.epam.jdi.light.driver.WebDriverFactory;
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.composite.WebPage;
+import io.github.epam.TestsInit;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+
+import static com.epam.jdi.light.common.TextTypes.TEXT;
+import static io.github.com.StaticSite.bsPage;
+import static io.github.com.pages.BootstrapPage.breadcrumb;
+import static io.github.epam.bootstrap.tests.BaseValidations.baseValidation;
+import static io.github.epam.states.States.shouldBeLoggedIn;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasItems;
+import static org.hamcrest.Matchers.is;
+
+public class BreadcrumbTests extends TestsInit {
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        bsPage.shouldBeOpened();
+    }
+
+    private static final String HOME = "Home";
+    private static final String HTML5 = "HTML 5";
+    private static final String BOOTSTRAP = "Bootstrap";
+    private static final String[] ITEMS_VALUES = new String[]{HOME, HTML5, BOOTSTRAP};
+
+    @Test
+    public void getTextTest() {
+        breadcrumb.itemns.has().size(ITEMS_VALUES.length);
+        breadcrumb.itemns.assertThat().values(TEXT, hasItems(ITEMS_VALUES));
+    }
+
+    @Test
+    public void getValueTest() {
+        breadcrumb.itemns.has().size(ITEMS_VALUES.length);
+        assertThat(breadcrumb.itemns.values().toArray(), is(ITEMS_VALUES));
+    }
+
+    @Test
+    public void getFirstItemTest() {
+        breadcrumb.itemns.first().has().value(HOME);
+        breadcrumb.itemns.first().has().text(HOME);
+    }
+
+    @Test
+    public void getCurrectItemTest() {
+        breadcrumb.itemns.last().has().value(BOOTSTRAP);
```

Comment:
itemns - possibly there is a typo, did you mean items ?

Reply:
yes, it'a a typo